test/unit/instance_config_interfaces_test.go

```diff
@@ -70,6 +80,11 @@ func TestInstanceConfigInterface_Create(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, "eth0", iface.Label)
 	assert.True(t, iface.Primary)
+
+	assert.Equal(t, "1234::5678/64", iface.IPv6.SLAAC[0].Range)
```

Comment:
In the future it'd be helpful if we could also assert on the raw request body made by unit test requests